plugins/gatsby-plugin-auto-index-pages/gatsby-node.js

```diff
@@ -150,13 +182,31 @@ exports.createPages = async ({ actions, graphql, reporter }, pluginOptions) => {
     locales.forEach(({ localizedPath }) => {
       const localizedSlug = path.join(`/${localizedPath}`, slug);
 
+      const localeDir = dir;
```

Comment:
Do we need to create a new variable for this?

Reply:
good question. i was getting some funky behavior when i was using a nested `visit()` buuut since i changed that this may not be necessary. will test it out